pkg/project/config.go

```diff
@@ -44,22 +44,25 @@ func (p *Config) ToFile() error {
 	return ioutil.WriteFile(filepath.Join(p.Dir, "nitric.yaml"), b, 0644)
 }
 
-// Assume the project is in the currentDirectory
-func ConfigFromFile() (*Config, error) {
-	wd, err := os.Getwd()
-	if err != nil {
-		return nil, err
+// ConfigFromProjectPath - loads the config nitric.yaml file from the project path, defaults to the current working directory
+func ConfigFromProjectPath(path string) (*Config, error) {
```

Comment:
path is not the best name for a variable as it conflicts with the builtin "path" module. not used in the file yet.. but can be confusing